WordPress/gutenberg
# Remove the document outline from the sidebar

closes #6478

**Testing instructions**

 - Add a post with multiple headings
 - Check that the table of contents is not showing up in the sidebar.
